### PR TITLE
make the LTS statuses in the table for Current clearer

### PR DESCRIPTION
**Pull request by Fishrock123**
I've always found "Not Available" a strange way to say "None" in this table.

### CODE REVIEW

**Comment by richardlau** [comment]
I think in this case `N/A` meant "not applicable" since neither v5 nor v7 are LTS releases.

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@
   <td>2019-04-18</td>
 </tr>
 <tr>
-  <td>N/A</td>
+  <td>None</td>
```
Comment (addaleax) [inline README.md]: tiny suggestion: I think `No LTS` might be even clearer :)

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@
   <td>2018-04-01</td>
 </tr>
 <tr>
-  <td>N/A</td>
+  <td>End-of-Life</td>
```
Comment (richardlau) [inline README.md]: Since this is in a `LTS Status` column, `End-of-Life` may be misinterpreted as suggesting this was previously LTS (it never was).

**Comment by Fishrock123** [comment]
Hmmm, looking again, maybe the v5 commit should be dropped?

**Pull request merged**